onmt/decoders/transformer.py

```diff
@@ -148,7 +178,9 @@ def __init__(self, num_layers, d_model, heads, d_ff,
             [TransformerDecoderLayer(d_model, heads, d_ff, dropout,
              attention_dropout, self_attn_type=self_attn_type,
              max_relative_positions=max_relative_positions,
-             aan_useffn=aan_useffn)
+             aan_useffn=aan_useffn,
+             full_context_alignment=full_context,
```

Comment:
Why two different names?

Reply:
Aims to shorten the sentence of line 169 and meet the requirement of flake (79).